src/authentication/index.js

```diff
@@ -546,6 +546,28 @@ Authentication.prototype.userInfo = function(accessToken, cb) {
     .end(responseHandler(cb, { ignoreCasing: true }));
 };
 
+/**
+ * Makes a call to the `/usernamepassword/challenge` endpoint
+ * and returns the challenge (captcha) if necessary.
+ *
+ * @method getChallenge
+ * @param {callback} cb
+ */
+Authentication.prototype.getChallenge = function(cb) {
+  assert.check(cb, { type: 'function', message: 'cb parameter is not valid' });
+
+  if (!this.baseOptions.state) {
+    return cb();
+  }
+
+  var url = urljoin(this.baseOptions.rootUrl, 'usernamepassword', 'challenge');
```

Comment:
I don't see this challenge endpoint documented anywhere, only [/mfa/challenge](https://auth0.com/docs/api/authentication#challenge-request). We should wait for that before adding this to the SDK.

Reply:
good point

Reply:
this is an internal endpoint, right? we don't document these publicly